Wox.Infrastructure/Logger/Log.cs

```diff
@@ -72,7 +72,7 @@ public static void Error(string message)
         public static void Exception(string message, System.Exception e)
         {
 #if DEBUG
-            throw e;
+            //throw e;
```

Comment:
why?

Reply:
lack of necessary diligence :/

while debugging i got this exception: 
    
>0x80070005 - Unspecified error
>'Microsoft Corporation' violates pattern constraint of '\bms-resource:.{1,256}'.

so i commented out, tested and forgot to re-enable it